ms/MSOper/MSSummary.cc

```diff
@@ -320,419 +320,164 @@ void MSSummary::listMain (LogIO& os, Record& outRec, Bool verbose,
-			os.output().setf(ios::right, ios::adjustfield);
-			os.output().width(widthScan); os << lastscan;
-			os.output().width(widthLead);  os << "  ";
-			os.output().setf(ios::right, ios::adjustfield);
-			os.output().width(widthFieldId); os << lastfldids(0) << " ";
-			os.output().setf(ios::left, ios::adjustfield);
-			if (name.length()>20) name.replace(19,1,'*');
-			os.output().width(widthField); os << name.at(0,20);
-			os.output().width(widthnrow);
-			os.output().setf(ios::right, ios::adjustfield);
-			os << _msmd->nRows(MSMetaData::BOTH, arrid, obsid, lastscan, lastfldids(0));
-			if (_listUnflaggedRowCount) {
-                          ostringstream xx;
-                          xx << std::fixed << setprecision(2)
-                             << _msmd->nUnflaggedRows(MSMetaData::BOTH, arrid, obsid, lastscan, lastfldids(0));
-                          os.output().width(widthNUnflaggedRow);
-                          os << xx.str();
-                        }
-//			os.output().width(widthInttim); os << meanIntTim;
-			os.output().width(widthLead);  os << "  ";
-                        showContainer (os.output(), spw, ", ", "[", "]");
-			os.output().width(widthLead); os << "  ";
-
-			std::map<uInt, Double> intToScanMap = _msmd->getAverageIntervalsForScan(lastScanKey);
-			os << "[";
-			for (
-				std::set<uInt>::const_iterator iter=spw.begin();
-				iter!=spw.end(); iter++
-			) {
-				if (iter!=spw.begin()) {
-					os << ", ";
-				}
-				os << intToScanMap[*iter];
-			}
-			os << "] ";
-			// The Obsmode column can be empty only report them if it is not
-			//String obsMode = "";
-			//if (obsModes.size() > (unsigned int) 0) {
-			//	obsMode=obsModes(laststids(0));
-			//}
-			//os << obsMode;
-			//os << endl;
-			set<String> intents = _msmd->getIntentsForScan(lastScanKey);
-			if (intents.size() > 0) {
-				os << intents;
-			}
-			os << endl;
-		}
-		if(fillRecord  && (recLength < maxRecLength)){
-			Record scanRecord;
-			Record subScanRecord;
-			String scanrecid=String("scan_")+String::toString(lastscan);
-			if(outRec.isDefined(scanrecid)){
-				scanRecord=outRec.asrwRecord(scanrecid);
-				outRec.removeField(scanrecid);
-			}
-
-			subScanRecord.define("BeginTime", btime/C::day);
-			subScanRecord.define("EndTime", etime/C::day);
-			subScanRecord.define("scanId", lastscan);
-			subScanRecord.define("FieldId", lastfldids(0));
-			subScanRecord.define("FieldName", name);
-			subScanRecord.define("nRow", thisnrow);
-			subScanRecord.define("IntegrationTime", meanIntTim);
-			subScanRecord.define("SpwIds", Vector<Int>(spw.begin(), spw.size(), 0));
-			scanRecord.defineRecord(String::toString(subsetscan), subScanRecord);
-			if(!outRec.isDefined(scanrecid)){
-				outRec.defineRecord(scanrecid, scanRecord);
-			}
-			subsetscan=0;
-			++recLength;
-		}
-
-		// post to logger
-		if (verbose) os << LogIO::POST;
-
-		// push OBS/ARR iteration
-		obsarriter.next();
-	} // end of OBS/ARR iteration
+    ArrayKey aryKey;
```

Comment:
Please keep the tab/space formatting the same within a file. Somehow your editor indented with spaces here, while the rest of the file is indented with tabs.